tests/sv/cover_HassCloseCover.yaml

```diff
@@ -1,6 +1,40 @@
 language: sv
 tests:
-- sentences: []
-  intent:
-    name: HassCloseCover
-    slots: {}
+  - sentences:
+      - "Stäng sovrumsgardinen"
+      - "Dra ner sovrumsgardinen"
+      - "Dra igen sovrumsgardinen"
+      - "Stäng igen sovrumsgardinen"
+      - "Veckla ner sovrumsgardinen"
+      - "Veckla igen sovrumsgardinen"
+      - "Tillslut sovrumsgardinen"
+    intent:
+      name: HassCloseCover
+      slots:
+        name: cover.bedroom_curtain
+
+  - sentences:
+      - "Stäng garagedörren"
+      - "Dra igen garagedörren"
+    intent:
+      name: "HassCloseCover"
+      slots:
+        device_class: "garage_door"
```

Comment:
I think that we can change the device_class to match: "garage" looks like it might be an issue in other languages also.
https://developers.home-assistant.io/docs/core/entity/cover/#device-classes